client/components/card/style.scss

```diff
@@ -13,61 +13,61 @@
 		margin-bottom: 16px;
 		padding: 24px;
 	}
+}
 
-	// Compact Card
-	&.is-compact {
-		margin-bottom: 1px;
+// Compact Card
+.isCompact {
```

Comment:
These rewrites to camelCase should be unnecessary.

I'm curious about how these selectors become less specific. They're no longer `.card.is-compact`, but just a single class. I prefer that! However, I wonder how much rewriting that would entail 🤔 

Do you know what the story is if we want to continue to use nested classes? Will css-modules handle that correctly?